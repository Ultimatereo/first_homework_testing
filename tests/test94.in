5
711241179350 804325051840 312050667429 -618120155288 -748141537313 -214894617006
-134356080820 162998814 -95848892000 -637169810736 -814095105516 -411400509737
-947085368229 -44284012582 503967277424 -267441147443 -153691140069 -326889338540
-433304029952 872025798909 946259876910 -447590749033 -91858423967 512131128043
-329078330662 -938430633640 -968554583551 671014264716 478751215970 -956884949450
