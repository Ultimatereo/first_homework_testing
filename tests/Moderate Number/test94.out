1.57126
-1.80224
0.187012
-6.94221
5.65712
